reverse_unique/fields.py

```diff
@@ -133,7 +133,7 @@ def get_extra_restriction(self, where_class, alias, related_alias):
         qs = remote_model.objects.filter(self.get_filters()).query
         my_table = self.model._meta.db_table
         rel_table = remote_model._meta.db_table
-        illegal_tables = set([t for t in qs.tables if qs.alias_refcount[t] > 0]).difference(
+        illegal_tables = set([t for t in qs.alias_map.keys() if qs.alias_refcount[t] > 0]).difference(
```

Comment:
Minor but I think the following would work

```suggestion
        illegal_tables = set([t for t in qs.alias_map if qs.alias_refcount[t] > 0]).difference(
```

Reply:
Hey, glad to see you! I was planning to do an update for some company internal purposes, but I can work a bit more on this to make this a bit better.

Would you be available for making a new release of this?